cla-backend-go/v2/cla_manager/service.go

```diff
@@ -522,6 +523,24 @@ func (s *service) InviteCompanyAdmin(contactAdmin bool, companyID string, projec
 
 }
 
+func (s *service) NotifyCLAManagers(notifyCLAManagers *models.NotifyClaManagerList) error {
+	// Search for Easy CLA User
+	log.Debugf("Getting user by ID: %s", notifyCLAManagers.UserID)
+	userModel, userErr := s.easyCLAUserService.GetUser(notifyCLAManagers.UserID)
+	if userErr != nil {
+		msg := fmt.Sprintf("Problem getting user by ID: %s ", notifyCLAManagers.UserID)
+		log.Warn(msg)
+		return ErrCLAUserNotFound
+	}
+
+	log.Debugf("Sending notification emails to claManagers: %+v", notifyCLAManagers.List)
+	for _, claManager := range notifyCLAManagers.List {
+		sendEmailToCLAManager(claManager.Name, claManager.Email, userModel.GithubUsername, notifyCLAManagers.Company, notifyCLAManagers.Company)
```

Comment:
The last parameter should be Project Name??

Reply:
👍